site/src/pages/docs/server-docservice.mdx

```diff
@@ -202,6 +202,8 @@ files into the specific location in the class path.
   </plugin>
   ```
 
+- Note that `.bin`, `.desc`, `.dsc`, `.pb` and `.protobin` are acceptable descriptor set file extensions.
+
```

Comment:
How about using `<Tip>` tag to emphasize it?

Reply:
Thanks! Like this?
![image](https://user-images.githubusercontent.com/8417787/140457311-73b60054-bf2a-4ea9-9afa-ac82ce580f58.png)

Reply:
Updated
![image](https://user-images.githubusercontent.com/8417787/140458995-70ead6e2-3cbf-4f46-86e0-f5a430274f61.png)

Reply:
Looks nice. :-)